.gitignore

```diff
@@ -21,6 +22,7 @@ shellcheck*
 check_shellcheck*
 test_spm.vocab
 test_spm.model
+custom*
```

Comment:
Do you need this?

Reply:
I was thinking about any custom training/decoding setup used for egs so they can be ignore, something like custom_training_my_rnn.yaml

Reply:
ok, but it may not be started from `custom*`. So, we may not need this.